plugins/catalog-backend/api-report.md

```diff
@@ -458,6 +459,12 @@ export type LocationSpec = {
   presence?: 'optional' | 'required';
 };
 
+// @public (undocumented)
+export function locationSpecToLocationEntity(
+  location: LocationSpec,
```

Comment:
Wondering if it actually makes more sense to wrap these arguments up in a function for extension later down the line if necessary. Didn't matter before as it wasn't public API, but now it will be, and thinking for the future! 🧙

Reply:
Do you mean to wrap location and parentEntity into an object, right? Just to confirm.

Makes sense.

Reply:
Yep indeed. Signature something like:

`function locationSpecToLocationEntity(opts: { location: LocationSpec, parentEntity?: Entity })` should be fine.

Reply:
done